client/components/fields/editor/base/dateTimeUIFramework.tsx

```diff
@@ -55,10 +57,17 @@ export class EditorFieldDateTimeUIFramework extends React.PureComponent<IProps>
                     error={error}
                     disabled={this.props.disabled}
                     dateFormat={appConfig.planning.dateformat}
-                    value={value == undefined ? null : new Date(value)}
+                    value={
+                        value == undefined
+                            ? null
+                            : {
+                                date: format(new Date(value), 'yyyy-MM-dd'),
+                                time: format(new Date(value), 'HH:mm'),
+                            }
+                    }
                     required={this.props.schema?.required}
                     onChange={(value) => {
-                        this.onChange(this.props.field, value);
+                        this.onChange(this.props.field, new Date(`${value.date} ${value.time}`));
```

Comment:
Time is optional, so if it's not set - you'll get invalid date. If I'm not wrong, `onChange` accepts ISO string of a date, not date itself.